.github/workflows/macos_tests.yml

```diff
@@ -5,11 +5,11 @@ on:
     inputs:
```

Comment:
Probably for a separate PR, but should we rename from `macos_tests` to, for example `xcode_tests` or `apple_tests`?

Reply:
Yeah I'm not sure - I hovered on `darwin_tests` too. I'll ask around.